<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>215</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mlSymMesh</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="get_source_pB">
      <property name="text">
       <string>Get source</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="source_lineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="get_target_pB">
      <property name="text">
       <string>Get target</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="target_lineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="bake_diff_pB">
      <property name="text">
       <string>Bake difference</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSlider" name="revert_to_base_slider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="revert_to_base_spinBox">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="revert_to_base_pB">
      <property name="text">
       <string>Revert selected to base</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>get_source_pB</tabstop>
  <tabstop>source_lineEdit</tabstop>
  <tabstop>get_target_pB</tabstop>
  <tabstop>target_lineEdit</tabstop>
  <tabstop>bake_diff_pB</tabstop>
  <tabstop>revert_to_base_slider</tabstop>
  <tabstop>revert_to_base_spinBox</tabstop>
  <tabstop>revert_to_base_pB</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>revert_to_base_slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>revert_to_base_spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>revert_to_base_spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>revert_to_base_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>